### DISCUSSION TITLE
ScriptReference/RenderTexture-height

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#363] [Notes]
# ScriptReference/RenderTexture-height

https://docs.unity3d.com/ScriptReference/RenderTexture-height.html

**Comment by HeliosDoubleSix** [comment]
If renderTexture has already been created then setting this will error: "Resizing of render texture that is loaded not supported!" you need to [`Release()`](https://docs.unity3d.com/ScriptReference/RenderTexture.Release.html) it first then you can set width/height again. This works for renderTextures that are unity asset files also.
